packages/app-aco/src/components/AdvancedSearch/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useRef } from "react";
+
+import { FormAPI } from "@webiny/form";
+import { DrawerContent } from "@webiny/ui/Drawer";
+import { observer } from "mobx-react-lite";
+// @ts-ignore
+import { useHotkeys } from "react-hotkeyz";
+import { Footer } from "./Footer";
+import { Header } from "./Header";
+
+import { QueryBuilder } from "~/components/AdvancedSearch/QueryBuilder/QueryBuilder";
+import { FieldRaw } from "~/components/AdvancedSearch/QueryBuilder/domain";
+
+import { DrawerContainer } from "./Drawer.styled";
+
+interface DrawerProps {
+    open: boolean;
+    onClose: () => void;
+    onSubmit: (data: any) => void;
+    fields: FieldRaw[];
+}
+
+export const Drawer = observer(({ open, onClose, fields, onSubmit }: DrawerProps) => {
```

Comment:
I think you can remove the `observer` HOC, since the `Drawer` itself is never reading from an observable object.